server/routes/creatingLicences/handlers/additionalLicenceConditionInput.ts

```diff
@@ -34,6 +36,20 @@ export default class AdditionalLicenceConditionInputRoutes {
 
     await this.licenceService.updateAdditionalConditionData(licenceId, conditionId, req.body, user)
 
+    if (req.file) {
+      // Check if there was a file upload in this POST request - only one allowed - the exclusion zone(outOfBoundFilename)
+      if (req.file.fieldname === 'outOfBoundFilename') {
```

Comment:
If the `req.file` is passed through the class-transformer correctly in the validation middleware, I dont think this check is needed, since the class-transformer will throw away anything which isn't defined in the class

Reply:
I don't like this bit either and open to suggestions. Separate validator sounds good, so will take a look.

Reply:
A separate validator is now used, which checks for a file upload and validates the field name, file name, mime type and size > 0 bytes.